lib/fortnox/api/validators/edi_information.rb

```diff
@@ -0,0 +1,16 @@
+require "fortnox/api/validators/base"
+
+module Fortnox
+  module API
+    module Validator
+      class EDIInformation
+
+        extend Fortnox::API::Validator::Base
+
+        using_validations do
+          # No validations needed, in other words always valid.
```

Comment:
Good idea. Then we would need some kind of super class for there two to guarantee that the API will stay the same over time, right?